lib/Command/ACL.php

```diff
@@ -90,32 +92,30 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			!$input->getArgument('path') &&
 			!$input->getArgument('permissions') &&
 			!$input->getOption('user') &&
+			!$input->getOption('team') &&
 			!$input->getOption('group')
 		) {
 			$this->printPermissions($input, $output, $folder);
-		} elseif ($input->getOption('manage-add') && ($input->getOption('user') || $input->getOption('group'))) {
-			$mappingType = $input->getOption('user') ? 'user' : 'group';
-			$mappingId = $input->getOption('user') ?: $input->getOption('group');
+		} elseif ($input->getOption('manage-add') && ($input->getOption('user') || $input->getOption('group') || $input->getOption('team'))) {
+			[$mappingType, $mappingId] = $this->convertMappingOptions($input);
 			$this->folderManager->setManageACL($folder['id'], $mappingType, $mappingId, true);
-		} elseif ($input->getOption('manage-remove') && ($input->getOption('user') || $input->getOption('group'))) {
-			$mappingType = $input->getOption('user') ? 'user' : 'group';
-			$mappingId = $input->getOption('user') ?: $input->getOption('group');
+		} elseif ($input->getOption('manage-remove') && ($input->getOption('user') || $input->getOption('group') || $input->getOption('team'))) {
+			[$mappingType, $mappingId] = $this->convertMappingOptions($input);
 			$this->folderManager->setManageACL($folder['id'], $mappingType, $mappingId, false);
 		} elseif (!$input->getArgument('path')) {
 			$output->writeln('<error><path> argument has to be set when not using --enable or --disable</error>');
 			return -3;
 		} elseif (!$input->getArgument('permissions')) {
 			$output->writeln('<error><permissions> argument has to be set when not using --enable or --disable</error>');
 			return -3;
-		} elseif ($input->getOption('user') && $input->getOption('group')) {
-			$output->writeln('<error>--user and --group can not be used at the same time</error>');
+		} elseif ((int)(bool)$input->getOption('user') + (int)(bool)$input->getOption('group') + (int)(bool)$input->getOption('team') > 1) {
```

Comment:
Uhm what? :see_no_evil: 
Is there no way to write this in a simpler and cleaner way?

Reply:
:D 
This is the simpler way